docs/installation.rst

```diff
@@ -318,6 +318,11 @@ which contains Troubleshooting sections.
 
 At this point, any Toil script can be run on the distributed AWS cluster following instructions in :ref:`runningAWS`.
 
+Finally, if you wish to tear down the cluster and remove all its data
+permanently, cgcloud allows you to do so without logging into the AWS web interface:
```

Comment:
This line should end with `::` so Sphinx will format this as a code block like the other CGCloud examples.